rust/processor/src/processors/fungible_asset_processor.rs

```diff
@@ -571,7 +571,11 @@ pub async fn parse_v2_coin(
             // This is because v1 events (deposit/withdraw) don't have coin type so the only way is to match
             // the event to the resource using the event guid
             let mut event_to_v1_coin_type: EventToCoinType = AHashMap::new();
-
+            // When coinstore is deleted we have no way of getting the mapping but hoping that there is
+            // only 1 coinstore deletion by owner address. This is a mapping between owner address and deleted coin type
+            // This is not ideal as we're assuming that there is only 1 coinstore deletion by owner address, this should be
+            // replaced by an event (although we still need to keep this mapping because blockchain)
```

Comment:
`CoinStoreDeletion` will replace `CoinEventHandleDeletion` and include CoinStore https://github.com/aptos-labs/aptos-core/pull/15730

Reply:
Great. I'm not using the events now so will just wait for that event to be on testnet and add the patch.